app/javascript/pages/browse/ContactIcons.vue

```diff
@@ -12,7 +12,7 @@ import TagList from 'components/TagList'
 const iconNameMapping = {
   email: 'envelope',
   text: 'mobile-alt',
-  call: 'phone',
+  phone: 'phone'
```

Comment:
I have no real preference what we name things, but I saw `phone` as a field in the database and not `call`, so I changed this temporarily so that it would work

This mapping can be long — we can put every synonym we can think of, if we want to go that far

Reply:
ah! makes sense! this is the lookup vs the dropdown